ParseException: unexpected token in list

Passing a list to a macro does not parse correctly

e.g.

```
HIGHLIGHT(["item1", "item2"], RED)
```

```
Caused by: com.lootfilters.lang.ParseException: unexpected token in list (token=Token{type=EXPR_END,value=)})
        at com.lootfilters.lang.TokenStream.expectStringList(TokenStream.java:225)
        at com.lootfilters.lang.Parser.parseItemNameRule(Parser.java:231)
        at com.lootfilters.lang.Parser.parseRule(Parser.java:205)
        at com.lootfilters.lang.Parser.lambda$parseMatcher$0(Parser.java:136)
        at com.lootfilters.lang.TokenStream.walkExpression(TokenStream.java:143)
        at com.lootfilters.lang.Parser.parseMatcher(Parser.java:111)
        at com.lootfilters.lang.Parser.parse(Parser.java:65)
        at com.lootfilters.LootFilter.fromSource(LootFilter.java:46)
        at com.lootfilters.LootFiltersPlugin.loadFilter(LootFiltersPlugin.java:275)
        at com.lootfilters.LootFiltersPlugin.startUp(LootFiltersPlugin.java:162)
        at net.runelite.client.plugins.PluginManager.startPlugin(PluginManager.java:429)
        ... 14 common frames omitted
```

I was able to get it working locally with this change (not sure if its the correct way to do it)
https://github.com/roundshoe/loot-filters/commit/9b7fbee282244df555b69fed5559e9f112550ca1#diff-883d4785d62a3c46ec7a597b3af52920cc62eb297557f2b95bb1b66b6b6e82e7R195-R197

--- a/src/main/java/com/lootfilters/lang/TokenStream.java
+++ b/src/main/java/com/lootfilters/lang/TokenStream.java
@@ -192,6 +192,9 @@ public class TokenStream {
             } else if (head.is(Token.Type.EXPR_START)) {
                 var nestedExpr = expr.take(Token.Type.EXPR_START, Token.Type.EXPR_END, true);
                 current.addAll(nestedExpr.getTokens());
+            } else if (head.is(Token.Type.LIST_START)) {
+                var nestedExpr = expr.take(Token.Type.LIST_START, Token.Type.LIST_END, true);
+                current.addAll(nestedExpr.getTokens());
             } else {
                 current.add(expr.take());
             }

